### DISCUSSION TITLE
MongoDB connections from AWS Lambda  using mangum as adapter

### DISCUSSION THREAD

**Discussion by himalacharya** [Kludex/mangum#227] [Q&A]
```
from fastapi import FastAPI
from mangum import Mangum

app = FastAPI()

@app.get("/")
def read_root():
   
    return {"Hello": "World"}


@app.get("/items/{item_id}")
def read_item(item_id: int, q: str = None):
    return {"item_id": item_id, "q": q}

handler = Mangum(app)
```
With reference to example, I have used in my project. For my project, I am using MongoDb but faces new connection for every time lambda function is invoked. I looked in Mongodb documentation (https://docs.atlas.mongodb.com/best-practices-connecting-from-aws-lambda/) that connection to Mongodb server should be outside lambda function.

Using example provided in documentation how to define client outside lambda function? All of my Database connection and query is above this 
`handler = Mangum(app)`

**Comment by njarma** [comment]
Have you found a solution?
